multi-node/aws/build

```diff
@@ -15,7 +15,8 @@ if [ -n "$(git diff --shortstat 2> /dev/null | tail -n1)" ]; then
 fi
 
 
-
+echo "Restoring deps"
+godep restore
```

Comment:
This should not be necessary. The `go build` command below overrides `GOPATH` to place the workspace earlier in the search path than your existing `GOPATH`.